Helper/OmiseHelper.php

```diff
@@ -292,4 +293,40 @@ public function createInvoiceAndMarkAsPaid($order, $chargeId, $isCapture = true)
         $invoice->setTransactionId($chargeId)->pay()->save();
         return $invoice;
     }
+
+    /**
+     * This method checks and return TRUE if $paymentMethod is
+     * offered by our Omise payment plugin otherwise returns false.
+     *
+     * @param string $paymentMethod
+     *
+     * @return boolean
+     */
+    public function isOmisePayment($paymentMethod)
```

Comment:
would it be better to pull lists from `isPayableByImageCode` and `isOffsitePayment` and then add credit card? so we don't have multiple lists to have to maintain

Reply:
It would be better. Thanks.